frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListsActionsBar/CreateNewCodeListModal/CreateNewCodeListModal.test.tsx

```diff
@@ -29,12 +29,14 @@ describe('CreateNewCodeListModal', () => {
     expect(codeListNameInput).toHaveTextContent('');
   });
 
-  it('renders the code list editor without content', async () => {
+  it('renders the code list editor without code list', async () => {
```

Comment:
```suggestion
  it('renders the code list editor without codelist', async () => {
```

Reply:
I think "code list" in two words is correct here. Otherwise we also need to change the casing in the code.

Reply:
I think "code list" in two words is correct here. Otherwise we also need to change the casing in the code.